scripts/create-kubestellar-demo-env.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+
+# The formal instructions accompaning this script can be found at:
+# https://docs.kubestellar.io/release-0.23.1/direct/get-started/
```

Comment:
I think it is best to use a relative reference, rather than introduce another place that requires editing for every release.

Reply:
I going to remove it to reduce confusion.